sections/escrow/components/StakingRewardsTab/TabContent.tsx

```diff
@@ -60,7 +61,7 @@ const TabContent: FC<TabContentProps> = ({
 					onClick={onSubmit}
 					variant="primary"
 					size="lg"
-					disabled={transactionState !== Transaction.PRESUBMIT || !!gasEstimateError}
+					disabled={transactionState !== 'unsent' || !!gasEstimateError}
```

Comment:
can we put that into a variable maybe?

Reply:
its a typescript limited string, so its impossible to put in bad values. works extremely similar to an enum.